ckanext/datastore/backend/postgres.py

```diff
@@ -434,8 +434,18 @@ def _where_clauses(
                 sa.column(field),
                 ','.join(f":{p}" for p in placeholders)
             ))
+            if fields_types[field] == 'text':
+                # pSQL can do int_field = "10"
+                # but cannot do text_field = 10
+                # this fixes parity there.
+                value = [str(v) for v in value]
```

Comment:
a generator expression here saves creating and throwing away an intermediate list
```suggestion
                value = (str(v) for v in value)
```